src/test/java/COSE/RegressionTest.java

```diff
@@ -51,6 +65,8 @@
             "Examples/sign-tests",
             "Examples/sign1-tests",
             "Examples/RFC8152",
+            "Examples/rsa-pss-examples",
+            // "Examples/X509",
```

Comment:
Don't include the X.509 or anima tests here.  Which means that there is probably some other clean up in this file that can be done.